x-pack/plugins/fleet/server/services/setup/managed_package_policies.ts

```diff
@@ -61,14 +167,11 @@ async function getPackagePoliciesNotMatchingVersion(
   soClient: SavedObjectsClientContract,
   pkgName: string,
   pkgVersion: string
-): Promise<PackagePolicy[]> {
-  return (
-    await packagePolicyService.list(soClient, {
-      page: 1,
-      perPage: 1000,
-      kuery: `${PACKAGE_POLICY_SAVED_OBJECT_TYPE}.package.name:${pkgName} AND NOT ${PACKAGE_POLICY_SAVED_OBJECT_TYPE}.package.version:${pkgVersion}`,
-    })
-  ).items;
+) {
+  return packagePolicyService.fetchAllItems(soClient, {
```

Comment:
`fetchAllItems` is an [async function](https://github.com/elastic/kibana/blob/13972565d7804e9583e68b0abdf9573cc36a4627/x-pack/plugins/fleet/server/services/package_policy.ts#L2127), did you omit the `await` on purpose or was it forgotten?